src/scripts.js

```diff
@@ -254,7 +254,7 @@ function dropYear(dates) {
       if (value === 0) {
         circle.setText('');
       } else {
-        circle.setText(`${activity.returnNumStepsDay(date)} steps`);
```

Comment:
This function looks like it was replaced accidentally, and we will need to put back in the num Steps function.